File: archive/azure-intro/python/01-iac/code/04-updating-your-infrastructure/step4.py
import pulumi
from pulumi_azure import core, storage

# Create an Azure Resource Group
resource_group = core.ResourceGroup('resource_group')

account = storage.Account('storage',
                          resource_group_name=resource_group.name,
                          account_tier='Standard',
                          account_replication_type='LRS')

container = storage.Container('mycontainer',
                        name = 'files',
                        storage_account_name = account.name)

pulumi.export('account_name', account.name)

File: archive/azure-intro/csharp/01-iac/code/03-provisioning-infrastructure/step4.cs
using Pulumi;
using Azure = Pulumi.AzureNative;
using Pulumi.AzureNative.Resources;

class MyStack : Stack
{
    public MyStack()
    {
        // Add your resources here
        // Create an Azure Resource Group
        var resourceGroup = new ResourceGroup("myrgroup");
        // Export the resource group name
        this.resourcegroupname = resourceGroup.Name;
    }

    // Add Outputs here
    [Output("resourcegroup_name")] public Output<string> resourcegroup_name { get; set; }
}
